src/components/MyPage.js

```diff
@@ -110,7 +110,8 @@ class MyPage extends Component {
   signOut() {
     try {
       UserUtil.signOut((result, error) => {
-        AsyncStorage.removeItem('token', () => { Actions.login(); });
+        let keys = ['token', 'filter'];
```

Comment:
Reset filter when sign out? Why?
Didn't back-end side save user filter information and adjust it?